.travis.yml

```diff
@@ -1,9 +1,16 @@
 sudo: false
 language: node_js
-node_js:
-  - 4
-  - 5
-  - stable
+
+matrix:
+  include:
+  - node_js: 4
+    env: CXX=g++-4.8
+  - node_js: 6
+    env: CXX=g++-4.8
+  - node_js: stable
+    env:
+    - SAUCE=true
```

Comment:
why? I already set it up this was in a couple of repos, the node version shouldn't have any influence for this.